src/lib.rs

```diff
@@ -1150,30 +1157,30 @@ pub trait Itertools : Iterator {
     ///     vec![1, 2, 4],
     ///     vec![1, 3, 4],
     ///     vec![2, 3, 4],
-    ///     ]);
+    /// ]);
     /// ```
     ///
     /// Note: Combinations does not take into account the equality of the iterated values.
+    ///
     /// ```
     /// use itertools::Itertools;
     ///
     /// let it = vec![1, 2, 2].into_iter().combinations(2);
     /// itertools::assert_equal(it, vec![
-    ///     vec![1, 2], // Note: these are the same
```

Comment:
Did you mean to delete these comments? They were recently added in #355.

Reply:
My mistake, I'll re-add.